### PR TITLE
Changed index.php file to index.example.php. Changed documentation so…

### PR DESCRIPTION
**Pull request by elenakolevska**
Moved index.php to index.example.php and updated the documentation completely so it's more apt for using Vicky as a package.

### CODE REVIEW

**Comment by elenakolevska** [comment]
@l80 Please check this PR, I changed the readme quite a bit. I left the init.d script and the documentation for it in there, until we move it and it's accepted in the SlackBot package. Is that ok?

File: README.md [hunk 1]
```diff
@@ -6,38 +6,26 @@ This library can receive and parse data from JIRA webhook, convert it into easil
 It comes loaded with a few [default listeners](#default-listeners) for the most common scenarios, but you can [extend it](#customizing) with your own custom logic too.
```
Comment (kommuna) [inline README.md]: [default listeners](#default-listeners) is not working

File: README.md [hunk 2]
```diff
@@ -6,38 +6,26 @@ This library can receive and parse data from JIRA webhook, convert it into easil
...
+        "kommuna/vicky": "dev-master",
+    },
+```   
+and run `composer install`.
 
 #### 1. Config files  
 Copy `bot/config.example.php` to `/etc/SlackBot/config.php` and `client/config.example.php` to `/etc/vicky/config.php` and follow the instructions in them to set your values.
 
-Get your bot token [here](https://my.slack.com/services/new/bot).
+You can get your slack bot token [here](https://my.slack.com/services/new/bot).
```
Comment (kommuna) [inline README.md]: @lv-dev "You can get..." looks like you are talking with somebody. Let's write clear technical text ;)

File: README.md [hunk 3]
```diff
@@ -6,38 +6,26 @@ This library can receive and parse data from JIRA webhook, convert it into easil
...
-#### 2. Server/Virtual host
-Set up a server that will listen for JIRA webhooks, use this nginx and php-fpm config as an example:
-```
-server {
-        listen          80;
-        server_name     your.host.url;
-        root            .../vicky/src;
-        index           index.php;
-        try_files       $uri $uri/ /index.php?$query_string;
-        error_page  405 =200 $uri;
```
Comment (kommuna) [inline README.md:28]: i'm not sure that config set is right. But ok.

File: README.md [hunk 4]
```diff
@@ -6,38 +6,26 @@ This library can receive and parse data from JIRA webhook, convert it into easil
...
 #### 1. Config files  
 Copy `bot/config.example.php` to `/etc/SlackBot/config.php` and `client/config.example.php` to `/etc/vicky/config.php` and follow the instructions in them to set your values.
 
-Get your bot token [here](https://my.slack.com/services/new/bot).
+You can get your slack bot token [here](https://my.slack.com/services/new/bot).
 
-Configure Slack bot, for details see [this] (https://github.com/jclg/php-slack-bot).
+>IMPORTANT: Please make sure that you have the webserver and the slackbot running on different ports. The slackbot port is configurable and you can set it in the `/etc/slackBot/config.php` file. 
 
-#### 2. Server/Virtual host
```
Comment (kommuna) [inline README.md:19]: That nginx + php-fpm conf. it's simplest way to config that. 
but for me ok.

File: README.md [hunk 5]
```diff
@@ -51,35 +39,127 @@ To stop the service run `service slackbotservice stop` or `/etc/init.d/myservice
...
+    | Register listeners
+    |--------------------------------------------------------------------------
+    |
+    | We're providing a few default listeners that would make sense for most teams.
+    | To add your own check out the "Adding custom listeners" section
+    |
+    */
+    
+    /**
+     * Send message to user if an issue gets assigned to them
```
Comment (kommuna) [inline README.md]: Not to user but to the user slack channel

**Review by kommuna — COMMENTED** [review]
@lv-dev all is ok, but i added some comments, please check it.

File: README.md (hunk 4 above)
Comment (elenakolevska) [reply README.md:19]: I actually removed this section completely, cause now people are using it as a package and the server is not really Vicky's "concern" anymore. They would have to have a server set up for their app anyway, maybe even just as a route from a bigger app. Let me know if you think I should put it back.

File: README.md (hunk 2 above)
Comment (elenakolevska) [reply README.md]: 😸  Personally.. I actually like the human-to-human approach, but you're right, the spec should be technical. I changed this to "The slack bot token can be obtained here".
There are other instances of "you" though...  For example: 
>"If you need to handle specific JIRA situations in your own, specific ways, you can register your own listeners." 
How do you think I should change this so it sounds more technical? 

@l80

File: README.md (hunk 1 above)
Comment (elenakolevska) [reply README.md]: @l80  Both links weren't working actually :( That's cause I had removed those headings(paragraphs) and moved the text somewhere else. So now I just removed the links all together.

Comment (kommuna) [reply README.md]: ok

File: README.md (hunk 4 above)
Comment (kommuna) [reply README.md:19]: ok

File: README.md (hunk 3 above)
Comment (kommuna) [reply README.md:28]: ok

**Comment by kommuna** [comment]
README is complete! Thank you @lv-dev !

**Pull request merged**